airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -709,10 +714,10 @@ class CloudSQLDatabaseHook(BaseHook):  # noqa
            in the connection URL
     :type default_gcp_project_id: str
     """
-
     conn_name_attr = 'gcp_cloudsql_conn_id'
     default_conn_name = 'google_cloud_sql_default'
-    conn_type = 'gcpcloudsql'
+    conn_type = 'gcpcloudsqldb'
```

Comment:
Doc comment for this class still says:

```
Serves DB connection configuration for Google Cloud SQL (Connections
    of *gcpcloudsql://* type).
```

Does changing this type make sense? (I'm not familar with these hooks)

Reply:
It should stay like that for backwards compatibility. It makes sense because those are really posfgres/mysql connections but they need to have different set of parameters.

Reply:
The docs don't line up with what the class currently uses I mean.

Reply:
h yeah. Mistake in doc